pylint/checkers/refactoring/refactoring_checker.py

```diff
@@ -483,6 +483,11 @@ class RefactoringChecker(checkers.BaseTokenChecker):
             "value by index lookup. "
             "The value can be accessed directly instead.",
         ),
+        "R1737": (
+            "Consider directly using 'yield from' instead",
+            "use-yield-from",
+            "Emitted when yielding from a loop can be replaced by yielding from the iterator directly.",
```

Comment:
```suggestion
            "yielding from a loop is slower than yielding from the iterator directly.",
```
(? not sure if actually true all the time, but I suppose it's the case. At least it's going to be more elegant and understandable than the loop.) 

I know almost all our messages talk about the "when" it's raised, when the "why" it's raised is a lot more interesting.

Reply:
Yes, I had thought about stating possible perf improvements but was not sure whether to say "possibly faster" or just state it. I guess we can go with this because it certainly won't be any slower. Besides explaining it is faster I also mentioned the cleaner code part.

Reply:
Sometime it's surprising, it would be better to add something in related.rst Found https://copyprogramming.com/howto/in-python-is-faster-using-yield-from-than-yield-in-a-loop  it would be better if it's in depth and by someone like a python core dev but I didn't find this in a reasonable time.